Libraries/Opc.Ua.Server/NodeManager/MasterNodeManager.cs

```diff
@@ -459,6 +459,90 @@ public void RegisterNamespaceManager(string namespaceUri, INodeManager nodeManag
             }
         }
 
+        /// <summary>
+        /// Unregisters the node manager as the node manager for Nodes in the specified namespace.
+        /// </summary>
+        /// <param name="namespaceUri">The URI of the namespace.</param>
+        /// <param name="nodeManager">The NodeManager which no longer owns nodes in the namespace.</param>
+        /// <exception cref="ArgumentNullException">Throw if the namespaceUri or the nodeManager are null.</exception>
+        public void UnregisterNamespaceManager(string namespaceUri, INodeManager nodeManager)
+        {
```

Comment:
Are also the nodes removed from the address space ? I only see the rebuilding of the namespace manager array.

Reply:
The address space is untouched in this method (similar to `RegisterNamespaceManager`) as we are only informing the `MasterNodeManager` that we have/own nodes in the said namespace or not. 

I believe that calls to the actual Node Manager instance `DeleteAddressSpace` would be required for node removal.

Looking at the usages of `m_namespaceManagers`, it appears to be primarily used in support of `GetManagerHandle`